src/components/Modal/ExternalLinkWarningModalWrapper.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useEffect, useMemo, useState } from 'react'
+
+import ExternalLinkWarningModal from './ExternalLinkWarningModal'
+
+type WarningModalProps = {
+  isWarningModalOpen: boolean
+  onContinue: () => void
+}
+
+function openInNewTab(url: string) {
+  window?.open(url, '_blank')?.focus()
+}
+
+const WHITELIST = [
+  /^https:\/\/([a-zA-Z0-9]+\.)?decentraland\.org(\/[^\s]*)?$/g,
+  /^https:\/\/([a-zA-Z0-9]+\.)?decentraland\.vote(\/[^\s]*)?$/g,
+  /^https:\/\/([a-zA-Z0-9]+\.)?snapshot\.org(\/[^\s]*)?$/g,
+  /^https:\/\/([a-zA-Z0-9]+\.)?dcl\.gg(\/[^\s]*)?$/g,
+]
+
+const INITIAL_STATE: WarningModalProps = {
```

Comment:
I'd change this to have `isWarningModalOpen` and `href` instead of `onContinue`.